ui/admin/app/components/form/onboarding/index.hbs

```diff
@@ -0,0 +1,63 @@
+<Rose::Form class='full-width' @cancel={{@cancel}} @onSubmit={{this.atSubmit}}>
+  <div class='create-resources-form'>
+    <Hds::Form::TextInput::Field
+      name='targetAddress'
+      @type='text'
+      @value={{this.targetAddress}}
+      @isRequired={{true}}
+      @isInvalid={{this.targetAddressInValid}}
+      {{on 'input' (set-from-event this 'targetAddress')}}
```

Comment:
@ZedLi Delete `onfocusout` event handlers. Now form validation happens within `atSubmit` method.

Reply:
could we call this action`submit` for consistency?

Reply:
yeah rename the action name from `atSubmit` to submit, to be consistent with other submit actions in forms

Reply:
If that was the change you mean already done 😃